zap/src/main/java/org/zaproxy/zap/extension/brk/BreakpointsPanel.java

```diff
@@ -295,15 +295,12 @@ public void run() {
 
     private void saveColumnWidth(String prefix, int width) {
         if (width > 0) {
-            if (log.isDebugEnabled())
-                log.debug(
-                        "Saving preference "
-                                + prefnzPrefix
-                                + prefix
-                                + "."
-                                + PREF_COLUMN_WIDTH
-                                + "="
-                                + width);
+            log.debug(
+                    "Saving preference {} {}. {}={}",
```

Comment:
Remove spaces between arguments, same for the other log statements.

Reply:
The space after the dot should also be removed (it's a whole configuration key).

Reply:
Ack

Reply:
Okay that should be done now too.